layouts/_default/baseof.html

```diff
@@ -30,7 +30,7 @@
     <meta property="og:image:width" content="{{ $.Scratch.Get "ogImageWidth" }}">
     <meta property="og:image:height" content="{{ $.Scratch.Get "ogImageHeight" }}">
     <meta property="og:title" content="{{ $.Scratch.Get "ogTitle" | default "IRL Podcast" }}">
-    <meta property="og:description" content="{{ .Page.Params.ogDescription | default "Veronica Belmont explores real stories of life online – and real talk about how we can all keep the Internet healthy, weird and wonderful. IRL is an original podcast from Mozilla Corporation." }}">
+    <meta property="og:description" content="{{ .Page.Params.ogDescription | default "Manoush Zomorodi explores real stories of life online – and real talk about how we can all keep the Internet healthy, weird and wonderful. IRL is an original podcast from Mozilla Corporation." }}">
```

Comment:
Do we want to take this opportunity to add "the" before "Mozilla Corporation" for better grammar?